packages/interactivity/docs/1-getting-started.md

```diff
@@ -23,26 +23,38 @@ To get started with the Interactivity API, you can follow this [**Quick Start Gu
 We can scaffold a WordPress plugin that registers an interactive block (using the Interactivity API) by using a [template](https://www.npmjs.com/package/@wordpress/create-block-interactive-template) with the `@wordpress/create-block` command.
 
 ```
-npx @wordpress/create-block my-first-interactive-block --template @wordpress/create-block-interactive-template
+npx @wordpress/create-block@latest my-first-interactive-block --template @wordpress/create-block-interactive-template
 ```
 
-#### 2. Generate the build 
+> **Note**
```

Comment:
I'm OK with adding this clarification as long as we remove it as soon as this issue is handled by the [scripts package](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-scripts/) and no additional action is required from the user.

I would feel better if before merging these changes we create an issue to remove this part when conditions allow (and share a link to it in this PR as a comment).

I'm approving these changes nevertheless

Reply:
Agreed, I'll create that issue and keep an eye on it.

Reply:
@juanmaguitar [issue created](https://github.com/WordPress/gutenberg/issues/56936). 

Please feel free to assign it to me and I'll keep an eye on when the note can be removed.